pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.scijava</groupId>
         <artifactId>pom-scijava</artifactId>
-        <version>30.0.0</version>
+        <version>31.1.0</version>
```

Comment:
Could you also update the  <artifactId>mobie-io</artifactId> version, please? I think that it would fail to build with the old one.

Reply:
Done.